### DISCUSSION TITLE
<title> issue

### DISCUSSION THREAD

**Discussion by caraegan** [community/community#39759] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi iv downloaded git bash and it won't let me save a repository keeps saying it can't find the path I don't know if i have its setup wrong i need help it lets me mkdir but won't save a repository.

**Comment by ossanna16** [comment]
Hi there @caraegan and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
